pegasus/router.rb

```diff
@@ -292,6 +292,8 @@ def document(path)
         cache :document
       end
 
+      pass if request.post? && !@header['post']
```

Comment:
What kind of page does the user get in this case?  I think we should return an Invalid request http response.